src/ansi-c/expr2c.cpp

```diff
@@ -3625,8 +3621,8 @@ std::string expr2ct::convert_with_precedence(
       return convert_prob_uniform(src, precedence=16);
     else if(statement==ID_statement_expression)
       return convert_statement_expression(src, precedence=15);
-    else if(statement==ID_gcc_builtin_va_arg_next)
-      return convert_function(src, "gcc_builtin_va_arg_next");
+    else if(statement == ID_va_start)
+      return convert_function(src, CPROVER_PREFIX "va_start");
     else
```

Comment:
Does dumping still work with that change? Meaning do we still get a C program that gcc compiles?

Reply:
The changes (in this PR) to goto-instrument/goto_program2code.cpp take care of this.